torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2913,6 +2913,17 @@ def sample_inputs_diag(op_info, device, dtype, requires_grad, **kwargs):
 
     return samples + [vec_sample]
 
+def sample_inputs_diag_embed(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = ((S, S),)
+
+    def generator():
+        for shape in cases:
+            yield(SampleInput(make_arg(shape)))
+
+    return list(generator())
```

Comment:
Keeping it in this format to maintain consistency with all the OpInfos. In case there is any addition required to the cases, this keeps it flexible.